scenarios/detection/FAQ.md

```diff
@@ -38,6 +38,26 @@ Annotated object locations are required to train and evaluate an object detector
 
 When creating a new project in VOTT, note that the "source connection" can simply point to a local folder which contains the images to be annotated, and respectively the "target connection" to a folder where to write the output. Pascal VOC style annotations can be exported by selecting "Pascal VOC" in the "Export Settings" tab and then using the "Export Project" button in the "Tags Editor" tab.
 
+For mask (segmentation) annotation, an easy-to-use online tool is
+[Labelbox](https://labelbox.com/).
```

Comment:
This looks great, with the screenshots. Text is good. Only missing item is a quick description what is good/bad about labelbox, and what the alternative(s) are. This is directly related to the work you did comparing different segmentation UIs.

Reply:
I added a sentence for other alternatives, since my comparison of those tools is just a quick simple review.

Reply:
Thanks!!